lib/oban/plugins/reindexer.ex

```diff
@@ -60,9 +60,9 @@ defmodule Oban.Plugins.Reindexer do
       :name,
       :schedule,
       :timer,
-      :timeout,
       indexes: ~w(oban_jobs_args_index oban_jobs_meta_index),
-      timezone: "Etc/UTC"
+      timezone: "Etc/UTC",
+      timeout: 15_000
```

Comment:
Nitpick, this is purely to match other plugins:

```suggestion
      timeout: :timer.seconds(15)
```